### PR TITLE
POL-273 - Unsanctioned Apps with Managed Contract

### PR DESCRIPTION
**Pull request by dfrankel33**
### Description

initial release of Unsanctioned SaaS Apps with Existing Contract policy
 

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Comment by dfrankel33** [comment]
tested policy incident: https://governance.rightscale.com/org/27730/projects/121937/policy-incidents/5ebd7f377848180001401b22

File: saas/fsm/unsanctioned_apps_with_contract/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+- Updated Credential service and Endpoint
+
+## v1.1
+
+- Updated the metadata
+
+## v1.0
+
+- initial release
```
Comment (cdwilhelm) [inline saas/fsm/unsanctioned_apps_with_contract/CHANGELOG.md:13]: Isn't this the initial release

File: saas/fsm/unsanctioned_apps_with_contract/fsm-unsanctioned_with_contract.pt [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+      field "annualCost", jmes_path(col_item, "annualCost")
+    end
+  end
+end
+
+script "js_products", type: "javascript" do
+  parameters "num_items", "rs_org_id"
+  result "request"
+  code <<-EOS
+  console.log("ds_products "+rs_org_id)
```
Comment (cdwilhelm) [inline saas/fsm/unsanctioned_apps_with_contract/fsm-unsanctioned_with_contract.pt:64]: remove logging

File: saas/fsm/unsanctioned_apps_with_contract/CHANGELOG.md (hunk 1 above)
Comment (dfrankel33) [reply saas/fsm/unsanctioned_apps_with_contract/CHANGELOG.md]: it is.  i just copied the changelog and forgot to update :)

File: saas/fsm/unsanctioned_apps_with_contract/fsm-unsanctioned_with_contract.pt (hunk 2 above)
Comment (dfrankel33) [reply saas/fsm/unsanctioned_apps_with_contract/fsm-unsanctioned_with_contract.pt]: fixed

**Pull request merged**